ArucoDetectRos-Amir's_Changes.py

```diff
@@ -0,0 +1,349 @@
+#!/usr/bin/env python3
+from __future__ import division  #in order to be able to set the (1/30)
+"""
+Framework   : OpenCV Aruco
+Description : Calibration of camera and using that for finding pose of multiple markers
+Status      : Working
+References  :
+    1) https://docs.opencv.org/3.4.0/d5/dae/tutorial_aruco_detection.html
+    2) https://docs.opencv.org/3.4.3/dc/dbb/tutorial_py_calibration.html
+    3) https://docs.opencv.org/3.1.0/d5/dae/tutorial_aruco_detection.html
+"""
+
+from math import floor
+import numpy as np
+import time
+import rospy
+from cv_bridge import CvBridge
+from sensor_msgs.msg import Image
+from sensor_msgs.msg import CameraInfo
+from sensor_msgs.msg import Imu
+import tf
+from std_msgs.msg import UInt32
+from std_msgs.msg import String
+import message_filters
+import tf.transformations as tr
+import numpy as np
+import pickle
+import glob
+import cv2
+import cv2.aruco as aruco
+import csv
+import tf2_ros
+
+calib_path_param = '/home/makeruser/wifi-Project/Aruco_Tracker/images/for_calib/*.jpg'  ########## AMIR -> change these images and location
+aruco_dict_param = aruco.DICT_4X4_250 ########## AMIR -> this work for https://chev.me/arucogen/
+marker_length = 0.19 # meters
+
+# floor_ids = [101,102]
+# human_ids = [100, 105]
+floor_ids = [102, 103, 104]
+human_ids = [0, 101, 1, 100]
+
+####---------------------- CALIBRATION ---------------------------
+def calib_camera(calib_path=calib_path_param):
+    # termination criteria for the iterative algorithm
+    criteria = (cv2.TERM_CRITERIA_EPS + cv2.TERM_CRITERIA_MAX_ITER, 30, 0.001)
+
+    # prepare object points, like (0,0,0), (1,0,0), (2,0,0) ....,(6,5,0)
+    # checkerboard of size (7 x 6) is used
+    objp = np.zeros((6*7, 3), np.float32)
+    objp[:,:2] = np.mgrid[0:7, 0:6].T.reshape(-1, 2)
+
+    # arrays to store object points and image points from all the images.
+    objpoints = [] # 3d point in real world space
+    imgpoints = [] # 2d points in image plane.
```

Comment:
hard to understand the variable names `objpoints` and `imgpoints`, also not in conventions! Needs `_` as separator between words:
```suggestion
    real_world_points = [] 
    imgage_points = [] 
```